src/styles/theme.ts

```diff
@@ -32,7 +32,8 @@ const theme = {
   },
   radii: {
     full: '9999px',
-    small: '5px'
```

Comment:
We are naming sizes with sm, md, lg in other components, could we standardize it? @miguel-nascimento

Reply:
Yup, i will!